src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] for ages 0, 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for ages 14, 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for ages 15, 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  test('should return [0, 0] for ages 24, 24', () => {
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+  });
+
+  test('should return [0, 0] for ages 28, 28', () => {
+    expect(getHumanAge(28, 28)).toEqual([3, 2]);
+  });
```

Comment:
The test description 'should return [0, 0] for ages 28, 28' is incorrect. It should be 'should return [3, 2] for ages 28, 28' to match the expected output `[3, 2]`.